main.nf

```diff
@@ -736,35 +736,37 @@ process rseqc {
     """
 }
 
-
 /*
- * Step 4.1 Rseqc create BigWig coverage
+ * Step 4.1 Subsample the BAM files if necessary
  */
-
-process createBigWig {
-    tag "${bam.baseName - 'sortedByCoord.out'}"
-    publishDir "${params.outdir}/bigwig", mode: 'copy'
-
-    when:
-    !params.skip_qc && !params.skip_genebody_coverage
+bam_forSubsamp
+    .filter { it.size() > params.subsampFilesizeThreshold }
+    .map { [it, params.subsampFilesizeThreshold / it.size() ] }
```

Comment:
Nice!

Reply:
Do we need to bump the minimum version of nextflow to be able to use `it.size()`?

Reply:
Also, is `params.subsampFilesizeThreshold` defined in `nextflow.config` and documented?

Reply:
As far as I know this should work just fine 👍 

The `params.subsampFileSizeThreshold` is defined in the `nextflow.config` - docs have been updated now too 👍